torch/_dynamo/eval_frame.py

```diff
@@ -315,15 +435,19 @@ def __enter__(self):
         self.backend_cache_manager.__enter__()
         self.backend_ctx = self.extra_ctx_ctor()
         self.backend_ctx.__enter__()
-        self.dynamic_ctx = enable_dynamic(self.dynamic, self.export)
```

Comment:
Interesting, we've lost self.export entirely, I guess it's not needed anymore?

Reply:
yeah its weird, I can get rid of it, let me first investigate when it was removed/why it was added in the first place.